AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/data/transcript/TranscriptAlternativeTests.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  TranscriptAlternativeTests.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+@testable import AmazonChimeSDK
+import XCTest
+
+class TranscriptAlternativeTests: XCTestCase {
```

Comment:
Do we need this test?

Reply:
I don't think this test case (and other data model constructor tests) adds much value itself but felt this was a pattern to follow to keep coverage, and in case in the future we need to expand the functionality of these classes, we have a place to start with. Let me know what you recommend.